.github/workflows/checks.yml

```diff
@@ -28,3 +28,33 @@ jobs:
       - run: yarn
       - run: yarn run prettify:diff
       - run: yarn run lint
+  mobile-tests:
+    name: Mobile Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: true
+      - name: Install Yarn dependencies
+        run: yarn
+      - name: Build Yarn packages
+        run: yarn build --ignore docs
+      - name: Verify locales if using l10n/* branch
+        run: |
+          if [[ "${GITHUB_REF##*/}" != l10n/* ]]; then
+            # We want to check translations only when they are complete
+            echo "Skipping translations check on non l10n branches"
+            exit 0
+          fi
+          cd packages/mobile
+          yarn --cwd packages/mobile test:verify-locales
+      - name: Run mobile tests
+        run: |
+          mkdir -p test-results/jest
+          yarn --cwd packages/mobile test:ci --runInBand
+      - name: 'Upload Test Results'
+        uses: actions/upload-artifact@v2
+        with:
+          path: test-results
```

Comment:
Looks there's no test result, though it was already broken on CircleCI 😅 

CircleCI
<img width="1208" alt="Screenshot 2021-06-22 at 11 44 53" src="https://user-images.githubusercontent.com/57791/122903017-49dc8d00-d34f-11eb-8f3f-77324e06aa75.png">

GitHub Actions
<img width="763" alt="Screenshot 2021-06-22 at 11 39 20" src="https://user-images.githubusercontent.com/57791/122902880-26b1dd80-d34f-11eb-9870-d346d323b807.png">

Reply:
sounds good!

Reply:
Could add a `if: failure()` which will skip this step unless there is a failure. [Context and expression syntax for GitHub Actions](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#failure).